dockerfiles/examples/azure/storage/values.schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "dllStorage": {
+      "$ref": "#/definitions/storage"
+    },
+    "dataStorage": {
+      "$ref": "#/definitions/storage"
+    },
+    "daliStorage": {
+      "$ref": "#/definitions/storage"
+    }
+  },
+  "title": "Values",
+  "type": "object",
+  "definitions": {
+    "storage": {
+      "type": "object",
+      "required": [ "size" ],
+      "properties": {
+        "size": {
+          "type": "string"
```

Comment:
Is string the correct type?

Reply:
I think has to be because needs to take values like "3M", "3Mi", "3Gi"..

Reply:
Should possibly be something like:
    "io.k8s.apimachinery.pkg.api.resource.Quantity": {
      "oneOf": [
        {
          "type": "string"
        },
        {
          "type": "integer"
        }
      ]
    },

Reply:
unless there's a way to actually type check the format properly, I think string should suffice